<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesManager</class>
 <widget class="QMainWindow" name="NotesManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotesManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(215, 203, 169);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayoutContent">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageNotes">
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
        <item row="0" column="0">
         <widget class="QWidget" name="widgetNavigation" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayoutTopics"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAddTopic">
             <property name="text">
              <string>Add Topic</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayoutContent">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutContenControls">
            <item>
             <spacer name="horizontalSpacerControls">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSizeNormal">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>A</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <property name="fontSize" stdset="0">
               <number>11</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSizeLarge">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>A</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <property name="fontSize" stdset="0">
               <number>14</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSizeHuge">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>17</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>A</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <property name="fontSize" stdset="0">
               <number>17</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(224, 224, 224);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageLogin">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="2">
         <spacer name="horizontalSpacerRight">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditPassCode">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>42</pointsize>
           </font>
          </property>
          <property name="maxLength">
           <number>8</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacerLeft">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Enter passcode to unlock:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <spacer name="verticalSpacerBottom">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacerTop">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLock">
   <property name="text">
    <string>Lock</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
